zerver/tests/test_users.py

```diff
@@ -817,8 +816,6 @@ def test_create_user_with_multiple_streams(self) -> None:
                         acting_user=None,
                     )
 
-        self.assert_length(queries, 91)
-
         self.assert_length(cache_tries, 28)
```

Comment:
Just worth noting we should be able to add an `assert_cache_query_count` too. Which may suggest we should name this `assert_database_query_count` for clarity.